packages/oui-radio-group/README.md

```diff
@@ -0,0 +1,37 @@
+# Radio-group
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+```html
+<oui-radio-group
+  model="..."
+  name="...">
```

Comment:
No `on-change` method on the group itself ? If I want to know when the value has changed I need to add the on-change on each `oui-radio` ?

Reply:
done